ssl/statem/statem_lib.c

```diff
@@ -1415,8 +1415,8 @@ unsigned long ssl3_output_cert_chain(SSL_CONNECTION *s, WPACKET *pkt,
 }
 
 /*
- * Tidy up after the end of a handshake. In the case of SCTP this may result
- * in NBIO events. If |clearbufs| is set then init_buf and the wbio buffer is
+ * Tidy up after the end of a handshake.
+ * If |clearbufs| is set then init_buf and the wbio buffer is
```

Comment:
```suggestion
 * If |clearbufs| is set then init_buf and the wbio buffer are
```
Also, why do init_buf and wbio not have |decorations|?